### PR TITLE
Makes usage of Nar-Sie consistent

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
spellcheck: Nar-Sie's name has been revised to Nar'Sie.
/:cl:

### CODE REVIEW

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -461,8 +461,8 @@
 		P.clashing = FALSE
 
 /obj/item/toy/plush/narplush
-	name = "nar'sie plushie"
-	desc = "A small stuffed doll of the elder god Nar'Sie. Who thought this was a good children's toy?"
+	name = "Nar-Sie plushie"
+	desc = "A small stuffed doll of the elder god Nar-Sie. Who thought this was a good children's toy?"
```
Comment (81Denton) [inline code/game/objects/items/plushes.dm]: Should probably be elder goddess for both dolls (muh lore)

**Comment by ExcessiveUseOfCobblestone** [comment]
Thank you for using **HER** correct pronoun

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -461,8 +461,8 @@
 		P.clashing = FALSE
 
 /obj/item/toy/plush/narplush
-	name = "nar'sie plushie"
-	desc = "A small stuffed doll of the elder god Nar'Sie. Who thought this was a good children's toy?"
+	name = "Nar-Sie plushie"
```
Comment (coiax) [inline code/game/objects/items/plushes.dm]: I think this needs `\improper`, because you'd still talk about the Nar-Sie plushie.

**Comment by coiax** [comment]
Gonna be honest, I prefer the apostrophe spelling. Why can't we just have that?

**Review by ChangelingRain — COMMENTED** [review]
I always used Nar-Sie, which is thus my preference.

**Comment by ExcessiveUseOfCobblestone** [comment]
This is the battle of the cent'ury

**Comment by ExcessiveUseOfCobblestone** [comment]
She's a strong, independent woman living in a sexist world where people paint HER as the bad ~~guy~~girl!

**Comment by Barhandar** [comment]
Nar-Sie is fairly pissed off at all the people misgendering _it_ with extra foul implications on where the blood comes from.

Good, that makes the cult stronger.

**Pull request merged**